### ISSUE TITLE
PASCAL_VOC datasets

### ISSUE THREAD

**Issue by miaoshuyu** [timy90022/One-Shot-Object-Detection#4]
So sorry!  Can you share your  PASCAL_VOC datasets.

**Comment by MSiam** [comment]
Yes I am looking for the same please will definitely help to quickly reproduce your result

**Comment by leotam** [comment]
As per the README.md, instructions should be from this repo: https://github.com/rbgirshick/py-faster-rcnn#installation-sufficient-for-the-dem

I downloaded using this:
wget http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtrainval_06-Nov-2007.tar
wget http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCtest_06-Nov-2007.tar
wget http://host.robots.ox.ac.uk/pascal/VOC/voc2007/VOCdevkit_08-Jun-2007.tar

and then link it via the recommended command, something like: 
`ln -s /raid/VOCdevkit/ /home/data/VOCdevkit2007`
